source/locale/pt_BR/LC_MESSAGES/integrating-physical-devices.po

```diff
@@ -21,8 +21,9 @@ msgid "Integrating physical devices"
 msgstr "Integrando dispositivos físicos"
 
 #: ../../source/integrating-physical-devices.rst:6
+#, fuzzy
 msgid "Who is this for: developers"
-msgstr "Para quem é: desenvolvedores"
+msgstr "Para quem é: usuários"
```

Comment:
In English it is for `developers`, but in Portuguese it is for `users`, is that right?